### ISSUE TITLE
Speed up deployment updates

### ISSUE THREAD

**Issue by brollb** [deepforge-dev/deepforge#1959]
Deployment updates currently take a bit of time as tensorflow is installed using conda. It would be better to build a custom, docker image based on the pulled image and then updating them close to instantaneously.

**Comment by umesh-timalsina** [comment]
I am not sure I understand, do you mean changing entrypoint for conda installs because of #1561 and include this in another Docker image?

**Comment by brollb** [comment]
Basically. I was imagining just building it on the deployment machine (not pushing it to dockerhub) though I guess it could be useful to push to dockerhub under a different tag.

**Comment by umesh-timalsina** [comment]
Yeah, I would imagine building the docker image in the deployment VM (via SSH, essentially the Dockerfile and script to do a docker build would live in the repo can be done). This problem will also continue with tensorflow 2.0 in the current VM.
